modules/mono/glue/GodotSharp/GodotSharp/Core/Basis.cs

```diff
@@ -623,21 +624,31 @@ private void SetDiagonal(Vector3 diagonal)
         /// </summary>
         /// <param name="target">The position to look at.</param>
         /// <param name="up">The relative up direction.</param>
+        /// <param name="useModelFront">
+        /// If true, then the model is oriented in reverse,
+        /// towards the model front axis (+Z, Vector3.ModelFront),
+        /// which is more useful for orienting 3D models.
+        /// </param>
         /// <returns>The resulting basis matrix.</returns>
-        public static Basis LookingAt(Vector3 target, Vector3 up)
+        public static Basis LookingAt(Vector3 target, Vector3? up = null, bool useModelFront = false)
         {
+            up ??= new Vector3(0, 1, 0);
```

Comment:
```suggestion
            up ??= Vector3.Up;
```

Nitpicking but this could be slightly better.